src/main/kotlin/rys/ajaxpetproject/service/ChatService.kt

```diff
@@ -0,0 +1,18 @@
+package rys.ajaxpetproject.service
+
+import org.bson.types.ObjectId
+import rys.ajaxpetproject.model.MongoChat
+interface ChatService {
+    fun createChat(mongoChat: MongoChat): MongoChat
+
+    fun findChatById(id: ObjectId): MongoChat?
+
+    fun findAllChats() : List<MongoChat>?
+
+    fun updateChat(id: ObjectId, updatedMongoChat: MongoChat): MongoChat?
+
+    fun deleteChat(id: ObjectId): Boolean
+
+    fun deleteChats()
+
+}
```

Comment:
```suggestion
    fun deleteChats()
}
```

there should be no empty line before the last brace. fix in all places

Reply:
done